dev-client/src/util.ts

```diff
@@ -63,7 +63,7 @@ export const formatPercent = (value: number) => {
 };
 
 export const formatCoordinate = (value: number) => {
-  return value.toLocaleString(undefined, {
+  return value?.toLocaleString(undefined, {
```

Comment:
What does this change (and the one above it in `StaticMapView`)  have to do with site screen colors? I think it should either be explained in the PR or moved to a different PR.

Reply:
It's unrelated. I hit this error during debugging and didn't think it was worth a separate PR. I'll move it.